litestar/stores/redis.py

```diff
@@ -92,6 +97,7 @@ def with_client(
         username: str | None = None,
         password: str | None = None,
         namespace: str | None | EmptyType = Empty,
+        automatic_store_lifetime: bool = True,
```

Comment:
```suggestion
```

I don't think we need this option here; If we create the client, we should always close it properly